docs/userguide/assets/snippets/view-controls.html

```diff
@@ -0,0 +1,3 @@
+<button class="api-migration-details-show-tldr" type="button">Show TL;DR only</button>
```

Comment:
Is it possible to also include the JavaScript snippet in this HTML file?
```javascript
<script type="text/javascript">
const isApiMigrationGuide = true;
</script>
```

Reply:
The `view-controls.html` is included too early in the process of loading page, therefore I only put HTML elements in it, but not the code.

It is recommended to load the JavaScript in the end of an HTML file (the HTML page rendered from prefix.md) because the browser has to interpret the whole `<script>` (which may generate additional HTML contents in the place of `<script>`) before continuing to load the page.  Running scripts too early will make the page laggy.

Reply:
The performance difference should be insigifnicant. But anyway, this is not a big deal. I was thinking about how we need to include when creating a new migration page. It is very possible that at some point, we will have multiple migration pages for different versions (e.g. pre-1.0, 2.0, etc). But that also happens rarely -- so it doesn't matter.